samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -44,7 +44,7 @@ static struct bt_iso_chan_ops iso_ops = {
 };
 
 static struct bt_iso_chan_io_qos iso_tx_qos = {
-	.sdu = sizeof(uint32_t), /* bytes */
+	.sdu = BT_HCI_ISO_DATA_HDR_SIZE + (sizeof(uint32_t)), /* bytes */
```

Comment:
This does not seems correct in terms of the definition of `sdu`:
`/** Channel SDU. Maximum value is BT_ISO_MAX_SDU */`

and

https://github.com/zephyrproject-rtos/zephyr/blob/b8cdf9d0a6372ce72727ca3dc4fcc39fc9ec66d6/include/zephyr/bluetooth/iso.h#L66-L67

Why cant the required change be done internally instead of each samples?

Reply:
The ISO SDU contains the data header, so the 4/8 octets of data header is counted towards the SDU size. 

So the maximum SDU size is indeed 0xFFF, but the maximum data length in the SDU is 0xFFF - 4 or 8, depending on whether the timestamp is added. 

I would really like this to be handled internally, but I am unsure how to do that since the buffer where the SDU (including the timestamp) is allocated by the application.